plugins/lockToVote/hooks/useProposalVeto.tsx

```diff
@@ -72,12 +72,13 @@ export function useProposalVeto(proposalId: string) {
 
     signPermit(dest, value, deadline).then((sig) => {
       if (!sig) return;
```

Comment:
Is that an expected neutral flow? Or is it an error?

Reply:
I would consider a catch-all check like `if(!sig?.yParity) throw new Error(...)`

Reply:
I like the catch/all idea 👍🏻